### PR TITLE
BS-13307 split ui designer

### PR DESCRIPTION
**Pull request by rbioteau**
https://github.com/bonitasoft/bonita-studio-sp/pull/21

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.designer.war/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: UI Designer War Fragment
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.designer.war/META-INF/MANIFEST.MF]: might worth adding Community in name as it is specific to Community package?

File: bundles/plugins/org.bonitasoft.studio.designer.war/Retrieve UI Designer war.launch [hunk 2]
```diff
@@ -16,5 +16,5 @@
 <listAttribute key="org.eclipse.debug.ui.favoriteGroups">
 <listEntry value="org.eclipse.debug.ui.launchGroup.run"/>
 </listAttribute>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:/org.bonitasoft.studio.pagedesigner}"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:/org.bonitasoft.studio.designer.war}"/>
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.designer.war/Retrieve UI Designer war.launch]: we should rename the Launch configuration n order to mention that it is the community version

**Comment by apupier** [comment]
we will also need a new Launch configuration to retrieve the lib of the UI designer model, not blocking for the merge

File: bundles/plugins/org.bonitasoft.studio.designer.war/META-INF/MANIFEST.MF (hunk 1 above)
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.designer.war/META-INF/MANIFEST.MF]: It is not present in a sp build...

File: bundles/plugins/org.bonitasoft.studio.designer.war/Retrieve UI Designer war.launch (hunk 2 above)
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.designer.war/Retrieve UI Designer war.launch]: There is UI designer and UI desifner sp

Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.designer.war/Retrieve UI Designer war.launch]: ok

File: bundles/plugins/org.bonitasoft.studio.designer.war/META-INF/MANIFEST.MF (hunk 1 above)
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.designer.war/META-INF/MANIFEST.MF]: ok

**Comment by apupier** [comment]
oK to merge when CI will be green

**Pull request merged**